flink-kubernetes-operator-api/src/main/java/org/apache/flink/kubernetes/operator/api/validation/CrdCompatibilityChecker.java

```diff
@@ -93,7 +93,10 @@ protected static void checkObjectCompatibility(
                     // This field was removed from Kubernetes ObjectMeta v1 in 1.25 as it was unused
                     // for a long time. If set for any reason (very unlikely as it does nothing),
                     // the property will be dropped / ignored by the api server.
-                    if (!fieldPath.endsWith(".metadata.clusterName")) {
+                    if (!fieldPath.endsWith(".metadata.clusterName")
+                            // This field was removed from the pod templates when upgrading
+                            // from crd-generator 6.8 to 6.11, needs further assessment
+                            && !fieldPath.contains(".volumeClaimTemplate.spec.resources.claims")) {
```

Comment:
This field was removed from Kubernetes API: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2620-L2625